src/web/templates/admin/event_layout.html

```diff
@@ -82,15 +82,24 @@ <h4 class="w-100 px-2 fs-5 mt-2 text-center">{{ admin_event.name }}</h4>
     {% endfor %}
     {% if not admin_event_tab %}
         <div class="mx-auto" style="max-width: 500px;">
-            <h5 class="my-5 text-center">
-                {{ _('No public screens') }}
-            </h5>
-            <p class="text-center">
-                {{ _("There are no public screens available for this event.") }}
-            </p>
-            <p class="text-center">
-                {{ _("If you have an account you may connect by clicking on the <i>Profile</i> button in the menu.") }}
-            </p>
+            {% if client.can_view_public_screens %}
+                <h5 class="my-5 text-center">
+                    {{ _('No public screens') }}
+                </h5>
+                <p class="text-center">
+                    {{ _("There are no public screens available for this event.") }}
+                </p>
+                <p class="text-center">
+                    {{ _("If you have an account you may connect by clicking on the <i>Profile</i> button in the menu.") }}
+                </p>
+            {% else %}
+                <h5 class="my-5 text-center">
+                    {{ _('You have no permissions') }}
+                </h5>
+                <p class="text-center">
+                    {{ _('You can ask the organizer for permission to access the event screens.') }}
```

Comment:
```suggestion
                    {{ _('You can ask the administrator for permission to access the event screens.') }}
```
Not necessarly the organizer, I think we should stick to this vocabulary here